### DISCUSSION TITLE
Integration in and importer in the self hosted version

### DISCUSSION THREAD

**Discussion by akib1689** [makeplane/plane#5287] [Q&A]
In the cloud hosted version, These things are available. I there any roadmap i.e. when can we expect this feature to come to self hosted version? 

We, have been using JIRA from the start of our project, and now we want to migrate. So, It doesn't make any sense for us to shift to another cloud hosted version. Also due to political unrest and other stuff. we are going fully local hosted version. When I was looking for alternative, I found plane and the UI and other stuff of plane made most sense, But this Github sync and the JIRA importer is not available in the self hosted version. We are also considering taiga, But the UI in here looked good. So, is there any sort of roadmap when this features will be available in the self hosted version?

**Comment by marcungeschikts** [comment]
I already asked the Plane team, and they informed me that I will need to migrate to the Plane Pro license to get GitHub and Jira integration. I am currently using the Plane One self-hosted version.
![image](https://github.com/user-attachments/assets/d3c198ee-4c78-4ce9-97d8-42a2638075f6)

**Reply by akib1689** [reply]
Thank you for your clarification.

**Comment by akib1689** [comment]
Does sync works like JIRA? Currently, we create and maintain issues in jira and there is no issue in our github repository, we create the issue in JIRA, then the github with JIRA integration triggers on pull request and branch opening. Does plane can manage issues from separate repositories? All the repositories are in a same organization.

From the video in their documentation, all I understood is the issues are created both on Github and plane. Then the issue management is done in github plane just sync's it. I am not getting is if issues from multiple repositories can be managed in a single project or not.
